README.md

```diff
@@ -29,12 +29,16 @@ As data storage SQLite or PostgreSQL (recommended) can be used.
 
 ## Development
 
+### Server
+
 `go run .` or `gow -e=go,mod,html,css run .` using [gow](https://github.com/mitranim/gow)
 
-To build the CSS run:
+### React Frontend
+
+Go to `/frontend` and run:
 
-1. `npm install`
-2. `npm run css`
+1. `yarn install`
+2. `yarn dev`
```

Comment:
It would be nice to be able to run the React app like this, without rebuilding it on every change